src/openeo_test_suite/tests/general/openapi.json

```diff
@@ -0,0 +1,6651 @@
+{
```

Comment:
this should be done with a git submodule instead of copy-pasting this huge JSON

e.g. see https://github.com/Open-EO/openeo-test-suite/tree/main/assets  where the openeo-processes project is already included with a submodule

Reply:
I've included it as a submodule (from the openeo-api repo) now inside of the general folder, is there any preference for another location?